pkg/report/datafeeds/types.go

```diff
@@ -2,43 +2,46 @@ package datafeeds
 
 import (
 	"fmt"
+	"math/big"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
 )
 
-// This is ABI encoding - abi: "(bytes32 FeedID, bytes RawReport)[] Reports" (set in workflow)
-// Encoded with: https://github.com/smartcontractkit/chainlink/blob/develop/core/services/relay/evm/cap_encoder.go
```

Comment:
I'd leave these comments and scope out follow-up tickets. There is no point we have three (3) different encoders/decoders for the same thing.

The idea is that the same encoder/decoder that's used on the workflow side (doing the encodign) should be used on the WT side (DF processor doing the decoding).

Reply:
https://smartcontract-it.atlassian.net/browse/BCFR-1234